src/javascript/app/base/header.js

```diff
@@ -49,9 +51,38 @@ const Header = (() => {
         fnc_exit : ['exitFullscreen', 'webkitExitFullscreen', 'mozCancelFullScreen', 'msExitFullscreen'],
     };
 
+    const addHeaderReadyCallback = (callback) => {
+        if (is_header_ready) {
+            callback();
+        } else {
+            header_ready_callbacks.push(callback);
+        }
+    };
+
+    const triggerHeaderReadyCallbacks = () => {
+        is_header_ready = true;
+        header_ready_callbacks.forEach(callback => {
+            try {
+                callback();
+            } catch (error) {
+                // console.error('Header ready callback error:', error);
```

Comment:
Should we silence these errors?
```suggestion
                console.error('Header error:', error);
```